App/Reducers/LoginReducer.js

```diff
@@ -9,24 +9,20 @@ export const INITIAL_STATE = Immutable({
 })
 
 // login attempts
-const attempt = (state, action) => {
-  return Immutable({ ...state, attempting: true })
-}
+const attempt = (state, action) =>
+  Immutable(state).merge({ attempting: true })
```

Comment:
If we fix the issue with `autoHydrate()`, this should be able to say as:

`  state.merge({attempting: true})`